EIPS/eip-7612.md

```diff
@@ -170,15 +170,17 @@ No backward compatibility issues found.
 
 ## Security Considerations
 
-<!--
-  All EIPs must contain a section that discusses the security implications/considerations relevant to the proposed change. Include information that might be important for security discussions, surfaces risks and can be used throughout the life cycle of the proposal. For example, include security-relevant design decisions, concerns, important discussions, implementation-specific guidance and pitfalls, an outline of threats and risks and how they are being addressed. EIP submissions missing the "Security Considerations" section will be rejected. An EIP cannot proceed to status "Final" without a Security Considerations discussion deemed sufficient by the reviewers.
+1. Data Availability: During the transition period, nodes need to maintain both MPT and VKT structures. This ensures that historical state can still be accessed while new state changes are recorded in the VKT overlay.
```

Comment:
makes sense but not a security consideration. this is the what this EIP actually says